templates/miq-template-ext-db.yaml

```diff
@@ -473,6 +477,49 @@ objects:
   spec:
     dockerImageRepository: "${HTTPD_IMG_NAME}"
 - apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: "${HTTPD_SERVICE_NAME}-configs"
+  data:
+    redirects.conf: |
+      # The following redirects files must be included to handle most specific to least specific URLs.
+
+      ### Cockpit redirects
+      ProxyPreserveHost on
+      RequestHeader unset X-Forwarded-Proto
+      RequestHeader set X-Forwarded-Proto 'https' env=HTTPS
+      <LocationMatch "^/cws/cockpi(t[^/]+|t)?/socket$">
+        ProxyPassMatch "ws://${NAME}:9002/cws/cockpi$1/socket"
+      </LocationMatch>
+      ProxyPass /cws/ http://${NAME}:9002/cws/
+
+      ### API redirects
+      ProxyPass /api https://${NAME}/api
+      ProxyPassReverse /api https://${NAME}/api
+
+      ### UI redirects
+      RewriteCond %{REQUEST_URI} !^/ws
+      RewriteCond %{REQUEST_URI} !^/proxy_pages
+      RewriteCond %{REQUEST_URI} !^/saml2
+      RewriteCond %{REQUEST_URI} !^/api
+      RewriteCond %{DOCUMENT_ROOT}/%{REQUEST_FILENAME} !-f
+      RewriteRule ^/ https://${NAME}%{REQUEST_URI} [P,QSA,L]
+      ProxyPassReverse / https://${NAME}/
+
+      ### WebSocket redirects
+      ProxyPass /ws ws://${NAME}/ws
+      ProxyPassReverse /ws ws://${NAME}/ws
+
+      ProxyPreserveHost on
+      RequestHeader set X_FORWARDED_PROTO 'https'
+    logging.conf: |
```

Comment:
Why are these separate files? I was thinking we would just supply the entire config in one file.